### ISSUE TITLE
Printing

### ISSUE THREAD

**Issue by SinisterSoft** [gideros/gideros#47]
Scintilla has the ability to print.  It would be good if that was added to the editor (it uses Scintilla).

http://www.scintilla.org/ScintillaDoc.html#Printing

**Comment by ar2rsawseen** [comment]
You mean something like this:

http://pyqt.sourceforge.net/Docs/QScintilla2/classQsciPrinter.html

**Comment by SinisterSoft** [comment]
yes.  It should be one call with -1,-1 maybe as the parameter - linewrap might have to be set, but then again it might default to wrap (which is good).

**Comment by SinisterSoft** [comment]
I think printing is available in scintilla for QT, not sure.  It would be nice to be able to print from the editor.
